quattor/functions/filesystem.pan

```diff
@@ -49,6 +49,32 @@ function lvs_add = {
     SELF;
 };
 
+@documentation{
+    desc = Adds a list of logical volumes to a volume group with mutiple lv options. To replace function lvs_add which only sets size.
+    arg = volume group, list of logical groups long with optons.  size and raid type are set if defined.
+}
+function lvo_add = {
+    function_name = 'lvo_add';
```

Comment:
You can drop this line, I think it's historical. The FUNCTION global is the new way to do this.